### ISSUE TITLE
Project from fable-getting-started won't load

### ISSUE THREAD

**Issue by aspnetde** [ionide/ionide-vscode-fsharp#1380]
I followed the [steps provided](https://zaid-ajaj.github.io/the-elmish-book/#/chapters/fable/hello-world) by Zaid in his Elmish Handbook to set up a simple hello world application.

The following works:

```bash
git clone https://github.com/Zaid-Ajaj/fable-getting-started.git
cd fable-getting-started
npm install
npm run build
```

Fable compiles and loading `./fable-getting-started` in VSCode with Ionide 4.11.1 works.

However, npm is reporting 17 vulnerabilities right now when running `npm install`. When I run `npm audit fix`, it claims that 14 of those had been fixed. Fable would still compile, but Ionide does not load the project anymore.

<img width="690" alt="image" src="https://user-images.githubusercontent.com/1322584/84142511-35d03100-aa55-11ea-84db-c01bc155806e.png">

I can't say for sure it's related to the npm audit call, but that's how I could reproduce it.

--

I am on macOS 10.15.4, running netcore 3.1.300 and node 14.4.0.

**Comment by forki** [comment]
I think the repro is not related to npm. Here is how i can repro the ionide issue:

1) `git clone https://github.com/Zaid-Ajaj/fable-getting-started.git`
2) `cd fable-getting-started`
3) `code .`
4) navigate to `App.fs` and see that syntactical highlighting actually kicks in
5) look at `src/obj' and see that `project.assets.json` was created. this implies ionide has properly run  `dotnet restore` internally
6 ) add `open Browser.Dom` in `App.fs`
7) notice red squigglies. this implies ionide did not reload the project after `project.assets.json` was created
8) close VS Code
9) `code .`
10) navigate to `App.fs` and notice that red squigglies are gone and `open Browser.Dom` is proeprly detected as "unused"

**Comment by aspnetde** [comment]
Correct.

**Comment by forki** [comment]
@Krzysztof-Cieslak same happening when I add sln in root

**Comment by isaacabraham** [comment]
I think that this is a dup of #1377

**Comment by forki** [comment]
yes but with repro ;-)

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed in `4.12.1`

**Comment by forki** [comment]
Awesome. Will test tomorrow morning. Thanks a lot

Krzysztof Cieślak <notifications@github.com> schrieb am Do., 11. Juni 2020,
15:04:

**Comment by aspnetde** [comment]
> Should be fixed in 4.12.1

Working for me 👍 Thanks!

**Comment by forki** [comment]
It almost works. *almost*

![image](https://user-images.githubusercontent.com/57396/84474006-5df4a580-ac8a-11ea-85e0-c22a043d12e4.png)

so the fix makes it load the project properly, but it does not update the App.fs. Only after touching the App.fs the squiggly goes away.

So new repro:

1) `git clone https://github.com/Zaid-Ajaj/fable-getting-started.git`
2) `cd fable-getting-started`
3) `code .`
4) navigate to `App.fs` and see that syntactical highlighting actually kicks in
5 ) add `open Browser.Dom` in `App.fs`
6) close VS Code
7) nuke `obj` and `bin` folders
8) `code .`
9) notice how project loads properly but squiggly in `App.fs` remains

**Comment by Krzysztof-Cieslak** [comment]
@forki that's duplicate of #1374 , let's track it there.
